examples/GLVolumeItem.py

```diff
@@ -32,13 +30,21 @@ def psi(i, j, k, offset=(50,50,100)):
     th = np.arctan2(z, np.hypot(x, y))
     r = np.sqrt(x**2 + y**2 + z **2)
     a0 = 2
-    ps = (1./81.) * 1./(6.*np.pi)**0.5 * (1./a0)**(3/2) * (r/a0)**2 * np.exp(-r/(3*a0)) * (3 * np.cos(th)**2 - 1)
-    return ps
+    return (
+        (1.0 / 81.0)
+        * 1.0
+        / (6.0 * np.pi) ** 0.5
```

Comment:
The lonely `* 1.0` seems to be a little excessive in splitting up this equation.
Maybe let it at least have the company of term in the following line
`* 1.0 / (6.0 * np.pi) ** 0.5`

Reply:
i adopted the suggested change from sorcery; but you're right, that line is a bit too broken up